### ISSUE TITLE
argdown-cli - Cannot find module 'lodash/merge'

### ISSUE THREAD

**Issue by antonostrovsky** [christianvoigt/argdown#156]
In order to use the new workflow and export HTML from Markdown-Argdown file, I have installed argdown-cli.

The package manager installation appears to be successful, but when trying to run 'argdown', I get the following error:

```
internal/modules/cjs/loader.js:638
    throw err;
    ^

Error: Cannot find module 'lodash/merge'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:636:15)
    at Function.Module._load (internal/modules/cjs/loader.js:562:25)
    at Module.require (internal/modules/cjs/loader.js:690:17)
    at require (internal/modules/cjs/helpers.js:25:18)
    at Object.<anonymous> (C:\Users\AppData\Roaming\npm\pnpm-global\3\node_modules\.pnpm\registry.npmjs.org\@argdown\core\1.4.7\node_modules\@argdown\core\dist\plugins\DataPlugin.js:21:33)
    at Module._compile (internal/modules/cjs/loader.js:776:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:787:10)
    at Module.load (internal/modules/cjs/loader.js:653:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
    at Function.Module._load (internal/modules/cjs/loader.js:585:3)

```
Could it be that some module is missing from dependency list, or is it my system?

Thank you in advance - really excited to be using the Markdown-Argdown combo!

Windows 10
Package info@ @argdown/cli@1.4.8 | MIT | deps: 8 | versions: 13

**Comment by christianvoigt** [comment]
good catch! The strange thing is that my tests did not fail, so I did not realize it before the release.

I removed the lodash dependency to reduce the size of @argdown/core, but did not add a dependency to "lodash.merge". I will release the fix as soon as possible.

**Comment by christianvoigt** [comment]
should be fixed now. The reason the import did not fail in my tests was that lodash was actually installed through some devDependencies.

**Comment by antonostrovsky** [comment]
I can confirm this is fixed in 1.4.9! Thank you for correcting it so promptly
